### PR TITLE
Indicate whether platform is 32 or 64 bit in the download dialog

### PR DESCRIPTION
**Pull request by imrehg**
Connects to #17

### CODE REVIEW

**Comment by imrehg** [comment]
Not sure if `etcher.min.js` should be part of the pull request as well, but since it was checked in, I've included it.

File: static/js/lib/s3.js [hunk 1]
```diff
@@ -154,7 +154,18 @@ function prettifyFileName(file) {
...
+        case "x86":
+          platform = platform + ", 32-bit"
+          break;
+        case "x64":
+          platform = platform + ", 64-bit"
+          break;
+        default:
+          break;
+      }
+      return "(" + platform + ")"
```
Comment (craigmulligan) [inline static/js/lib/s3.js]: IMO it seems more natural to write it like: `Etcher for Linux x64  (64-bit)`, having two things in parenthesis seems weird.

**Review by craigmulligan — APPROVED** [review]
Thanks for this! Looks good, up to you on string layout 👍

File: static/js/lib/s3.js (hunk 1 above)
Comment (imrehg) [reply static/js/lib/s3.js]: Yup, it looks nicer indeed, pushing an updated version :)

**Pull request merged**